src/main/kotlin/org/jmailen/gradle/kotlinter/tasks/format/FormatWorkerAction.kt

```diff
@@ -73,27 +76,19 @@ abstract class FormatWorkerAction : WorkAction<FormatWorkerParameters> {
             },
         )
     }
+}
 
-    private fun formatKt(file: File, ruleSets: Set<RuleProvider>, onError: ErrorHandler) =
-        format(file, ruleSets, onError, false)
+private fun KtLintRuleEngine.formatKt(file: File, onError: ErrorHandler) =
+    format(file, onError)
 
-    private fun formatKts(file: File, ruleSets: Set<RuleProvider>, onError: ErrorHandler) =
-        format(file, ruleSets, onError, true)
+private fun KtLintRuleEngine.formatKts(file: File, onError: ErrorHandler) =
+    format(file, onError)
 
-    private fun format(file: File, ruleProviders: Set<RuleProvider>, onError: ErrorHandler, script: Boolean): String {
-        return KtLint.format(
-            KtLint.ExperimentalParams(
-                fileName = file.path,
-                text = file.readText(),
-                ruleProviders = ruleProviders,
-                script = script,
```

Comment:
Looks like ktlint now checks if the file is a script internally:
<img width="579" alt="image" src="https://user-images.githubusercontent.com/3951580/208132187-9f15b1d6-4b82-491a-ab1a-043790cebc59.png">

I missed that initially, looks like I can remove the kt/kts distinction from this file?